projects/packages/backup-helper-script-manager/src/class-helper-script-manager-impl.php

```diff
@@ -5,7 +5,11 @@
  * @package automattic/jetpack-backup
  */
 
-namespace Automattic\Jetpack\Backup;
+// After changing this file, consider increasing the version number ("VXXX") in the namespace, in order to ensure that
```

Comment:
Just this file, or all the files using the "Automattic\Jetpack\Backup\V0001" namespace?

Reply:
Thanks, updated in 712c9e9.